src/main/java/Task.java

```diff
@@ -0,0 +1,13 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) { //creating new object
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
```

Comment:
Try to minimize the usage lambda functions if possible as they make code less readable